File: TestProject2/Form1.cs
namespace SCO203
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        internal static void Login_Click(string username, string password)
        {
            throw new NotImplementedException();
        }

        public void Login_Click(object sender, EventArgs e)
        {
            string username = Username.Text;
            string password = Password.Text;
            if (username == "admin" && password == "admin")
            {
                MessageBox.Show("Login Successful");
            }
            else
            {
                MessageBox.Show("Invalid Username or Password");
            }
        }
    }
}

File: TestProject5/Form1.vb
Public Class Form1
    Public Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Public Sub Login_Click(sender As Object, e As EventArgs) Handles Login.Click
        If Username.Text = "admin" And Password.Text = "admin" Then
            MsgBox("Login Successful")
        Else
            MsgBox("Login Failed")
        End If
    End Sub
End Class
